Gruntfile.js

```diff
@@ -181,9 +181,33 @@ module.exports = function (grunt) {
 	];
 
 	grunt.initConfig(gruntConfig);
-
 	grunt.registerTask('compiler', ['curl:compiler', 'unzip:compiler']);
-	grunt.registerTask('all', ['copy', 'concat', 'closure-compiler']);
+	grunt.registerTask('all', ['copy', 'concat', 'requirejs', 'closure-compiler']);
+
+	grunt.loadTasks('spec/tasks');
+
+	var browsers = grunt.option('browsers') || 'default';
+  grunt.registerTask('test',
```

Comment:
> I do, but right now I just have bigger things to worry about than to fix an issue that is only visible in Github because it assumes tabs are 4 spaces instead of 2.

Fair enough.

> The rest of the library uses spaces, so this file is the exception.

Actually I think most of the library contains tabs. You are actually bringing the 2 spaces indent in the repo. Check the other files you modified/deleted.